docroot/themes/humsci/humsci_basic/patterns/horizontal-card/horizontal-card.html.twig

```diff
@@ -1,46 +1,46 @@
-{%- set attributes = attributes.addClass('hb-cmp-horizontal-card') -%}
+{%- set attributes = attributes.addClass('hb-cmp-card hb-cmp-card--horizontal') -%}
```

Comment:
To add multiple classes, use an array type syntax.
https://befused.com/drupal/twig-addclass